### PR TITLE
feat: add the filter for the job search page relates #45

### PR DESCRIPTION
**Pull request by naghamabuwarda**

### CODE REVIEW

**Review by shathakh — APPROVED** [review]

**Review by alishanti98 — APPROVED** [review]

**Review by mohammedagha27 — APPROVED** [review]

**Review by mohammedagha27 — APPROVED** [review]

File: client/src/components/filter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```
Comment (LinaYahya) [inline client/src/components/filter/index.tsx:1]: remove these commented lines, and try to fix errors

File: client/src/components/filter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  category: string,
+  changeCategory: (cate: any) => void,
+  priceChange: (data: any) => void,
+  price : number
+}
+
+function Filter({
+  category, changeCategory, priceChange, price,
+}: Props) {
+  console.log(category);
```
Comment (LinaYahya) [inline client/src/components/filter/index.tsx:20]:
```suggestion
```

File: client/src/components/filter/style.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  line-height: 15px;
+  color: #1C3879;
+
+}
+.categories{
+  font-family: 'Ubuntu';
+  font-style: normal;
+  /* font-weight: 100; */
+  /* font-size: 12px; */
+  /* line-height: 15px; */
```
Comment (LinaYahya) [inline client/src/components/filter/style.css:25]:
```suggestion
 
```

File: client/src/pages/JobsSearch/index.tsx [hunk 4]
```diff
@@ -24,8 +25,25 @@ const tablist: Array<TabListInt> = [{
...
+  const [price, setPrice] = useState<number>(0);
+  const [category, setCategory] = useState<string>('');
+  const changeCategory: React.ChangeEventHandler<HTMLInputElement> = (e) => {
+    setCategory(e.target.value);
+  };
+  const priceChange: React.ChangeEventHandler<HTMLInputElement> = (e) => {
+    setPrice(Number(e.target.value));
+  };
+  // console.log(category);
+  // console.log(price);
```
Comment (LinaYahya) [inline client/src/pages/JobsSearch/index.tsx:37]:
```suggestion

```

**Review by LinaYahya — COMMENTED** [review]
Linters to fix

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**